Get utc full year issue (#2348)
Codecademy/docs: content/javascript/concepts/dates/terms/getUTCFullYear/getUTCFullYear.md

---

> title: ".getUTCFullYear()"

Reviewer: ```suggestion
Title: '.getUTCFullYear()'
```

> description: "Returns the year of the specified date according to universal time. It extracts the four-digit year (e.g., 2023) from a given Date object, based on the Coordinated Universal Time (UTC) time zone."

Reviewer: ```suggestion
Description: 'Returns the year of the specified date according to universal time.'
```

Reviewer: Please update the remaining lines within the metadata section to use single quotes

Subjects:
  - "Web Development"
  - "Computer Science"
Tags:
  - "Date"
  - "Methods"
CatalogContent:

>   - "introduction-to-javascript"

Reviewer: ```suggestion
  - 'introduction-to-javascript'
  - 'paths/front-end-engineer-career-path'
```

---

> 
> ## Introduction

Reviewer: ```suggestion
```

Reviewer: We begin docs entries with the description not a header

> The `.getUTCFullYear()` method is a built-in function in JavaScript that returns the year of the specified date according to universal time. It extracts the four-digit year (e.g., 2023) from a given Date object, based on the Coordinated Universal Time (UTC) time zone.

Reviewer: ```suggestion
The **`.getUTCFullYear()`** method is a built-in function in JavaScript that returns the year of the specified date according to universal time. It extracts the four-digit year (e.g. 2023) from a given Date object, based on the Coordinated Universal Time (UTC) time zone.
```

This method is particularly useful when you need to retrieve the year component of a date regardless of the time zone or local system settings. It ensures consistent results regardless of the user's geographical location.

> ## Syntax

Reviewer: ```suggestion
## Syntax

```

Author: I don't understand what is the suggestion here...

Reviewer: It was to leave an empty line after the header

> ```js

Reviewer: ```suggestion
```pseudo
```

Date.getUTCFullYear();
```

> 

Reviewer: ```suggestion
We typically include a short sentence or two about the objects or parameters involved. Here it could be valuable to briefly describe a Date object.
```

## Example

> Consider the following example that demonstrates the usage of the `getUTCFullYear()` method:

Reviewer: ```suggestion
Consider the following example that demonstrates the usage of the `.getUTCFullYear()` method:
```

```js
const date = new Date('2023-06-06T10:30:00Z');
const year = date.getUTCFullYear();

> console.log(year);  // Output: 2023

Reviewer: this missing the closing ```

---
Title: '.getUTCFullYear()'
Description: 'Returns the year of the specified date according to universal time.'
Subjects:
  - 'Web Development'
  - 'Computer Science'
Tags:
  - 'Date'
  - 'Methods'
CatalogContent:
  - 'introduction-to-javascript'
  - 'paths/front-end-engineer-career-path'
---

The **`.getUTCFullYear()`** method is a built-in function in JavaScript that returns the year of the specified date according to universal time. It extracts the four-digit year (e.g. 2023) from a given `Date` object, based on the Coordinated Universal Time (UTC) time zone.

> This method is particularly useful when you need to retrieve the year component of a date regardless of the time zone or local system settings. It ensures consistent results regardless of the user's geographical location.

Reviewer 2: ```suggestion
This method is particularly useful when it's necessary to retrieve the year component of a date regardless of the time zone or local system settings. It ensures consistent results regardless of the user's geographical location.
```
We avoid the use of first- and second-person pronouns in Docs.

## Syntax

```pseudo
const date = new Date();

> date.getUTCFullYear();

Reviewer 2: ```suggestion
result = date.getUTCFullYear();
```

```

A `Date` object is used for working with dates and times. The object provides a way to create, manipulate, and format dates in various formats.

A `Date` object can be created by declaring a new variable with the `new Date()` constructor.

## Example

Consider the following example that demonstrates the usage of the `.getUTCFullYear()` method:

```js
const date = new Date('2023-06-06T10:30:00Z');
const year = date.getUTCFullYear();

> console.log(year); // Output: 2023

Reviewer 2: Rather than a comment in the code, can we have a ` ```shell ` block illustrating the output? Something like the following:

````
This example results in the following output:

```shell
2023
```
````
